flujo/application/core/policies/tree_search_policy.py

```
@@ -0,0 +1,873 @@
+                "tree_search_state",
+                "current_step",
+                "current_step_index",
+            }
+            merged = safe_merge_context_updates(context, winner_ctx, excluded_fields=excluded)
+            if merged is False:
+                try:
+                    merged_ctx = ContextManager.merge(context, winner_ctx)
+                    if isinstance(merged_ctx, PipelineContext):
+                        context = merged_ctx
+                except Exception:
+                    pass
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)